cmd/gaia/app/genesis_test.go

```diff
@@ -142,3 +142,24 @@ func TestNewDefaultGenesisAccount(t *testing.T) {
 	require.Equal(t, sdk.NewInt(1000), acc.Coins.AmountOf("footoken"))
 	require.Equal(t, sdk.NewInt(150), acc.Coins.AmountOf(bondDenom))
 }
+
+func TestGenesisStateSanitize(t *testing.T) {
```

Comment:
Should add a sub-test to make sure account numbers are being sorted correctly.

Reply:
addressed, thanks